cmd/run.go

```diff
@@ -166,6 +166,8 @@ func (o *runOption) preRunE(cmd *cobra.Command, args []string) (err error) {
 	case "http":
 		templateOption := runner.NewTemplateOption(o.reportTemplate, "json")
 		o.reportWriter = runner.NewHTTPResultWriter(http.MethodPost, o.reportDest, nil, templateOption)
+	case "grpc":
+		o.reportWriter = runner.NewGRPCResultWriter(o.reportDest)
```

Comment:
Prefer to check the value of `o.reportDest`.

Reply:
I think if we want to send a report to the grpc server, this variable should be an API on the corresponding grpc server. 
Do you mean I need to check if it is empty before passing it to the function🤔?

Reply:
I mean the variable `o.reportDest` is required. We can return an error to let users know they missed it.

Reply:
By the way, you can pass `o.context` into this function. So you don't need to create a new one.

See https://github.com/LinuxSuRen/api-testing/pull/431/files#diff-960009bb026ade66a155b9f92d7011cc78f516ac27c656618665600792a7af7fR57

Reply:
now, `grpcResultWriter` has a member variable `context`, and when creating it,  `o.Context` will be passed in.